### DISCUSSION TITLE
Looking for simple Langium based XML grammar that demonstrates the use of lexer modes

### DISCUSSION THREAD

**Discussion by dsmarhas** [eclipse-langium/langium#1365] [Q&A]
Beginner in Langium, I have been trying to create a Langium grammar that specifies  simple XML. However the playground does not allow me to create multiple lexer modes. In Xtext, I was able to use a custom lexer.  Not sure how to proceed. Thanks in advance for any guidance.

**Comment by msujew** [comment]
@dsmarhas We have a work-in-progress branch where I've tried to document the feature in more depth, see [here](https://github.com/eclipse-langium/langium-website/blob/ef14db1549b1ae177325cae024ab51a2225ab53b/hugo/content/guides/multi-mode-lexing.md). Basically, you are supposed to just use [Chevrotains lexer modes](https://chevrotain.io/docs/features/lexer_modes.html).
